packages/core/playwright.config.ts

```diff
@@ -7,13 +7,14 @@ const config: PlaywrightTestConfig = {
   testMatch: '*.e2e.ts',
   use: {
     baseURL: 'http://localhost:3333',
+    browserName: 'chromium',
```

Comment:
Would it not make sense to include other rendering engines? Like Webkit for Safari and Gecko for Firefox? I think these automated tests is a great place for catching inconsistencies between browsers, rather than you guys having to check all browsers.

Reply:
Think that makes sense, I think you can set it up something like the suggested change below.

Reply:
Don't know if this will impact the amount of snapshots though, but would be good to try out

Reply:
I agree, I think we can add more of those like Tim suggested in the comment below. My concern is how long it should take to run the test when committing too. Right now it is round 20sec on my Mac, with changes in this branch it goes down to 14sec. 

I want to optimize it so it does not take that much time. Maybe we need to consider running only test of components you work on currently and not entire system? Maybe entire system check should happen on opened PR instead and block merging if it is failing?

Reply:
I think you could move the running of tests to the github action only, and remove it from the pre-commit hook?

Reply:
we are not alone: https://github.com/microsoft/playwright/issues/15075